src/process_wrap.cc

```diff
@@ -211,6 +211,13 @@ class ProcessWrap : public HandleWrap {
     // options.stdio
     ParseStdioOptions(env, js_options, &options);
 
+    // options.windowsHide
+    Local<String> windows_hide_key = env->windows_hide_string();
+
+    if (js_options->Get(windows_hide_key)->IsTrue()) {
```

Comment:
This should ideally use the overload that takes a `Local<Context>` but OTOH, all the other options don't.

Reply:
@bnoordhuis I'll submit a follow up PR to switch all of the options to use the context.